app/server/api/example/[id].get.ts

```diff
@@ -0,0 +1,19 @@
+import { defineEventHandler } from 'h3'
```

Comment:
please remove all unused example stuff -> this is an actual project made with sidebase, actual projects would also remove / replace everything that is from sidebase but not needed